udev/65-multipath.rules

```diff
@@ -8,6 +8,9 @@ ACTION=="remove", ENV{DM_MULTIPATH_DEVICE_PATH}=="1",  GOTO="count_mpath"
 IMPORT{db}="DM_ACTION"
 ACTION=="change", ENV{DM_ACTION}=="PATH_REINSTATED", GOTO="count_mpath"
 ACTION=="change", ENV{DM_ACTION}=="PATH_FAILED", GOTO="count_mpath"
+ACTION=="add", PROGRAM=="/bin/bash -c '/sbin/dmsetup table -j %M -m %m | /bin/grep multipath'", ENV{MULTIPATH}="1"
+IMPORT{db}="MULTIPATH"
+ACTION=="add|remove", ENV{MULTIPATH}=="1", GOTO="count_mpath"
```

Comment:
Can lines 6 and 7  be removed now because of this?

Reply:
Ah sorry i see, that is matching is against another ENV variable.

Reply:
Yes, add/remove DM_MULTPATH_DEVICE_PATH is adding a sub device of the multipath